site/content/docs/5.3/components/local-navigation.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title: Local navigation
+description: Use local navigation to add a navigation that will nicely wrap on small viewports.
+group: components
+aliases:
+  - "/docs/components/local-navigation/"
+toc: true
+added: "5.3"
+---
+
+## Example
+
+Resize your browser to see the behavior of this component.
+
+{{< callout info >}}
+Since the component is supposed to be rendered on a full-size page, there are some slight rendering issues in the mobile viewport in the context of this documentation page.
+{{< /callout >}}
+
+{{< example >}}
+<nav class="local-nav" aria-label="Basic local navigation">
+  <button class="local-nav-button collapsed d-lg-none" type="button" data-bs-toggle="collapse" data-bs-target="#collapseLocalNav" aria-expanded="false" aria-controls="collapseLocalNav">
+    <span class="container-xxl">Shop</span>
+  </button>
+  <div id="collapseLocalNav" class="container-xxl local-nav-collapse d-lg-block collapse">
+      <div class="navbar navbar-expand-lg">
+        <ul class="navbar-nav w-100">
+          <li class="nav-item"><a class="nav-link" href="#"><div class="container-xxl">Discover</div></a></li>
```

Comment:
In this collapse navbar, shouldn't we use `<span class="container-xxl">` instead of `<div class="container-xxl">` everywhere? It's used like this within the `<button>` that triggers the collapse, and it seems to be more logical in the tree to have spans within an `<li>` > `<a>` rather than a `<div>`.